src/app/core/store-facade/store-facade.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import { select, Store } from "@ngrx/store";
+import { Observable } from "rxjs";
+import { map } from "rxjs/operators";
+import { IUserLogin, IUser, ICourse } from "../models/index";
+
+import * as fromApp from "../store/app.reducer";
```

Comment:
store/index
and in store index:
`import * as AuthActions from "./auth.actions";`
`import * as AuthSelectors from "./auth.selectors";`
`export { AuthActions , AuthSelectors  }`

it will let you use AuthActions and AuthSelectors in other files

Reply:
done!